src/pages/Search/SearchHoldReasonPage.tsx

```diff
@@ -25,12 +27,11 @@ type SearchHoldReasonPageProps = {
 function SearchHoldReasonPage({route}: SearchHoldReasonPageProps) {
     const {translate} = useLocalize();
 
-    const {currentSearchHash, selectedTransactions} = useSearchContext();
-    const {backTo = ''} = route.params ?? {};
+    const {currentSearchHash} = useSearchContext();
+    const {backTo = '', transactionID = ''} = route.params ?? {};
 
-    const selectedTransactionIDs = Object.keys(selectedTransactions);
     const onSubmit = (values: FormOnyxValues<typeof ONYXKEYS.FORMS.MONEY_REQUEST_HOLD_FORM>) => {
-        SearchActions.holdMoneyRequestOnSearch(currentSearchHash, selectedTransactionIDs, values.comment);
+        SearchActions.holdMoneyRequestOnSearch(currentSearchHash, [transactionID], values.comment);
```

Comment:
Won't this break bulk hold/unhold with multiple transactionIDs?

Reply:
Looks like it doesn't work if trying to hold from the search screen now, I get this response:

```json
{
    "jsonCode": 402,
    "message": "402 Missing transactionIDList",
    "onyxData": [],
    "requestID": "8c507fed5f4371d6-FRA"
}
```